avocado/utils/output.py

```
@@ -45,22 +43,23 @@ def display_data_size(size):
+
+        :param minimum: Minimum (initial) value on the progress bar
+        :type minimum: int
+        :param maximum: Maximum (final) value on the progress bar
+        :type maximum: int
+        :param width: Number of columns, that is screen width
+        :type width: int
+        :param title: Optional title to display with the progress bar
+        :type title: str
+        :raises AssertionError: When maximum is not greater than minimum
         """
         assert maximum > minimum
```

Ruff (0.12.2)

Use of `assert` detected

(S101)